app/config/config.go

```diff
@@ -30,72 +44,63 @@ type Exercise struct {
 }
 
 type Participant struct {
-	GithubUserName string
+	IntraLogin     string `json:"intra_login"`
+	GithubUserName string `json:"github_username"`
 }
 
-// Reads the participants list (json) from participantsListPath.
-//
-// Returns a slice of Participant structs containing the GitHub usernames of the participants.
-func NewParticipants(participantsListPath string) (participants []Participant, err error) {
-	data, err := os.ReadFile(participantsListPath)
+var C Config
+
+func init() {
+	err := godotenv.Load("../.env")
 	if err != nil {
-		return nil, fmt.Errorf("unable to read config file %s: %v", participantsListPath, err)
+		logger.Warning.Printf(".env file not found, this is expected in the GitHub Actions environment, this is a problem if you are running this locally\n")
```

Comment:
Thinking of this maybe we should remove this error message?
Not using `.env` files but actually defining environment variables on their own is also a valid use case.

Also when this will be run as a docker container, the container will probably not use an `.env` file but just define the variables themselves.